### PR TITLE
Testing deploy action

### PR DESCRIPTION
**Pull request by phyzical**

### CODE REVIEW

**Comment by phyzical** [comment]
ah thats right it has to be merged to be tested...

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ name: Build Docker Image
 on:
   push:
   schedule:
-    - cron: "0 0 1 * *"
+    - cron: '0 0 1 * *'
```
Comment (phyzical) [inline .github/workflows/deploy.yml:5]:
```suggestion
    - cron: "0 0 1 * *"
```

**Review by waldofouche — APPROVED** [review]
GL

**Pull request merged**